cosmos/dbt/graph.py

```diff
@@ -75,8 +76,8 @@ class DbtGraph:
     def __init__(
         self,
         project: DbtProject,
-        exclude: list[str] | None = None,
-        select: list[str] = None,
+        exclude: Optional[list[str]] = None,
```

Comment:
An alternative from doing like this is to rely on the fact we're doing;
```
from __future__ import annotations
```

And just do:

```suggestion
        exclude: list[str] | None = None,
```

Other parts of the code use this style, it may be worth to be consistent.